envoy/upstream/load_balancer.h

```diff
@@ -158,9 +168,15 @@ class LoadBalancerFactory {
   virtual ~LoadBalancerFactory() = default;
 
   /**
-   * @return LoadBalancerPtr a new load balancer.
+   * @return LoadBalancerPtr a new worker local load balancer.
+   * TODO(wbpcode): remove this method in the future and used the new method below.
    */
   virtual LoadBalancerPtr create() PURE;
+
+  /**
+   * @return LoadBalancerPtr a new worker local load balancer.
+   */
+  virtual LoadBalancerPtr create(LoadBalancerParams params) PURE;
```

Comment:
Why this is necessary? 
If the `load_balancing_policy` is enabled, all Lb type will be treated as `ThreadAwareLoadBalancer`. But for Lb type like `least request`, thread local `priority_set` and `local_priority_set` are necessary. So, we must add this `params` to help the factory to create these types of Lb.